testsuite/openshift/objects/auth_config/sections.py

```diff
@@ -180,23 +198,27 @@ def add_uma(self, name, endpoint, credentials_secret, **common_features):
 
 
 class ResponseSection(Section):
-    """Section which contains response configuration"""
+    """Section which contains response configuration. todo envoyDynamicMetadata"""
```

Comment:
If we are leaving dynamic metadata implementation for the future PR, we should somehow mark wristband tests as they are currently raising an error

Reply:
Added pytest skip